autowarefoundation/autoware.universe
# fix(accel brake map calibrator): correction of forgotten support for util files change

## Description

calc_utils.py is added calibration_method argument in the following PR.
It was necessary to change new_accel_brake_map_server.py as same.
I fix this script.

https://github.com/autowarefoundation/autoware.universe/pull/1255

And I changed how to get path parameter to get from ros parameter.

how to test
- launch autoware 
- launch accel brake map calibrator
  - ros2 launch accel_brake_map_calibrator accel_brake_map_calibrator.launch.xml rviz:=false
- execute below script and check accel_brake_map.svg

```
import rclpy
from rclpy.node import Node
import time

from tier4_external_api_msgs.srv import GetAccelBrakeMapCalibrationData

class MinimalClientAsync(Node):

    def __init__(self):
        super().__init__('minimal_client_async')
        self.cli = self.create_client(GetAccelBrakeMapCalibrationData, '/api/external/get/accel_brake_map_calibrator/data')
        while not self.cli.wait_for_service(timeout_sec=1.0):
            self.get_logger().info('service not available, waiting again...')
        self.get_logger().info("service is created")

    def send_request(self):
        if not self.cli.service_is_ready():
            self.get_logger().error(
                "service is unavailable, request: {}".format(str(request))
            )
            return
        self.get_logger().info("service is call")
        req = GetAccelBrakeMapCalibrationData.Request()

        future = self.cli.call_async(req)
        print(future)
        self.get_logger().info("service is calling")
        rclpy.spin_until_future_complete(self, future)
        response = future.result()

        f = open('accel_brake_map.svg', 'wb')
        f.write(response.graph_image)
        f.close()

	
def main():
    rclpy.init()

    minimal_client = MinimalClientAsync()
    minimal_client.send_request()


    minimal_client.destroy_node()
    rclpy.shutdown()


if __name__ == '__main__':
    main()
```



## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
